app/controllers/application_controller.rb

```diff
@@ -1,5 +1,17 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  # session checking for customer available or not
+  def session_checking
+    if current_admin.nil?
```

Comment:
Nested if?

Can we refactor this  using && operator and making it single line!